From d183d7b75656fbc6d3cd38cc2a7263ca7b0f8409 Mon Sep 17 00:00:00 2001
From: Rafael Beims <rafael.beims@toradex.com>
Date: Tue, 5 Sep 2023 15:27:57 -0300
Subject: [PATCH] Remove REGULATORY_IGNORE_STALE_KICKOFF flag

This flag was removed by commit wifi: cfg80211: fix regulatory
disconnect with OCB/NAN upstream and was not yet updated on the latest
NXP driver release.

Signed-off-by: Rafael Beims <rafael.beims@toradex.com>
---
 mlinux/moal_sta_cfg80211.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/mlinux/moal_sta_cfg80211.c b/mlinux/moal_sta_cfg80211.c
index 71daa62..00efb5b 100755
--- a/mlinux/moal_sta_cfg80211.c
+++ b/mlinux/moal_sta_cfg80211.c
@@ -10005,12 +10005,14 @@ woal_register_cfg80211(moal_private *priv)
 	}
 #endif
 #if CFG80211_VERSION_CODE >= KERNEL_VERSION(3, 19, 0)
+#if CFG80211_VERSION_CODE < KERNEL_VERSION(6, 1, 0)
 	/*REGULATORY_IGNORE_STALE_KICKOFF: the regulatory core will _not_ make sure
 	 * all interfaces on this wiphy reside on allowed channels. If this flag
 	 * is not set, upon a regdomain change, the interfaces are given a grace
 	 * period (currently 60 seconds) to disconnect or move to an allowed
 	 * channel.*/
 	wiphy->regulatory_flags |= REGULATORY_IGNORE_STALE_KICKOFF;
+#endif
 #endif
 
 	memset(&priv->phandle->country_code, 0,
-- 
2.39.2

